test/testkit/matchers/trace/ptrace_utils.go

```diff
@@ -1,30 +1,61 @@
 package trace
 
 import (
+	"go.opentelemetry.io/collector/pdata/pcommon"
 	"go.opentelemetry.io/collector/pdata/ptrace"
 
 	"github.com/kyma-project/telemetry-manager/test/testkit/matchers"
 )
 
+type FlatTrace struct {
```

Comment:
Please add a documentation comment here similar to what we have for `FlatMetric`